pages/auth/setup-security-questions.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from "react";
```

Comment:
@thiessenp-cds I added a temporary page here for the redirect to land on when your logging in (if you haven't already set them up).   We can change pending how we setup 

https://github.com/cds-snc/platform-forms-client/issues/2417

We have 2 cases -- existing users and new registration.